public/app/features/alerting/unified/MuteTimings.tsx

```diff
@@ -21,8 +21,10 @@ const MuteTimings = () => {
   const config = currentData?.alertmanager_config;
 
   const getMuteTimingByName = useCallback(
-    (id: string): MuteTimeInterval | undefined => {
-      const timing = config?.mute_time_intervals?.find(({ name }: MuteTimeInterval) => name === id);
+    (id: string, fromTimeIntervals: boolean): MuteTimeInterval | undefined => {
+      // merge both fields mute_time_intervals and time_intervals to support both old and new config
+      const time_intervals = fromTimeIntervals ? config?.time_intervals ?? [] : config?.mute_time_intervals ?? [];
+      const timing = time_intervals?.find(({ name }: MuteTimeInterval) => name === id);
```

Comment:
nit: optional chaining not needed here, time_intervals should be an array always :)

Reply:
ohhhh yes, I forgot this one